src/app/drivingLicence/fields.js

```diff
@@ -72,22 +72,35 @@ module.exports = {
     validate: [
       "required",
       "date",
-      { type: "after", arguments: [new Date().toISOString().split("T")[0]] },
+      { type: "after", arguments: [new Date().toISOString().split("T")[0]] }
     ],
   },
   drivingLicenceNumber: {
     type: "text",
     journeyKey: "drivingLicenceNumber",
     validate: [
-      "required"
+      "required",
+      { type: "maxlength", arguments: [30] },
+      { type: "minlength", arguments: [30] },
+      { type: "regexDrivingLicence", fn: (value) => value.match(/^(?=.{16}$)[A-Za-z]{1,5}9{0,4}[0-9](?:[05][1-9]|[16][0-2])(?:[0][1-9]|[12][0-9]|3[01])[0-9](?:99|[A-Za-z][A-Za-z9])(?![IOQYZioqyz01_])\w[A-Za-z]{2}$/) }
+    ],
+    classes: "govuk-input--width-10",
+  },
+  dvaLicenceNumber: {
+    type: "text",
+    journeyKey: "dvaLicenceNumber",
+    validate: [
+      "required",
+      { type: "regexDrivingLicence", fn: (value) => value.match(/^[0-9]{8}$/) }
```

Comment:
we may need to make this field dependant on the licenceIssuerDependent field in the same way it was done for issue date/number fields

Reply:
Added.